### PR TITLE
Update network graph

### PR DESCRIPTION
**Pull request by kcarmichael08**
Version(s):

- Merge to `rhacs-docs`
- Cherry pick to `rhacs-docs-4.1`

[Issue](https://issues.redhat.com/browse/ROX-17269)

[Link to docs preview
](https://61772--docspreview.netlify.app/openshift-acs/latest/operating/manage-network-policies.html#network-graph-20-docs)

QE review:
- [x] QE has approved this change. (**ACS has no QE, approved by SME**)

Additional information:

- This PR is to remove "2.0 preview" language, remove old network graph modules, and update docs with changes to the UI that were made since release 4.0.

### CODE REVIEW

**Review by dvail — COMMENTED** [review]
My comments here are all based on the fact that in the current network simulator, in both 4.0 and 4.1, generating network policies will always create policies for _all_ of the deployments in the selected cluster.

Work ongoing in 4.2 is fixing this in various ways:
- We are in the process of making it more clear exactly how network policies are generated, and the scope of what the generated policies cover, but changing wording, etc. in the UI.
- We are updating the policy generation functionality so that network policies will be generated for all deployments that match the user's selected scope. For this purpose, the selected scope is all deployments that match the Cluster/Namespace/Deployment dropdowns at the top of the page.
- There are no plans to make the time range dropdown or the generic filters affect the scope of generated policies. This is something that is being discussed for the future but is not intended to target the 4.2 release.

**Comment by kcarmichael08** [comment]
Approved via slack by Saif, via gh by David.

File: modules/configure-cidr-blocks-ng20.adoc [hunk 1]
```diff
@@ -3,18 +3,18 @@
...
 
 .Procedure
-. In the {product-title-short} portal, navigate to *Network Graph (2.0 preview)*, and then select *Manage CIDR Blocks*. You can perform the following actions:
+. In the {product-title-short} portal, navigate to *Network Graph*, and then select *Manage CIDR Blocks*. You can perform the following actions:
 * Toggle *Auto-discovered CIDR blocks* to hide auto-discovered CIDR blocks in the network graph.
 +
 [NOTE]
 ====
-When you hide the auto-discovered CIDR blocks, the auto-discovered CIDR blocks are hidden for all clusters, not only for the selected cluster on the top bar in the network graph.
+When you hide the auto-discovered CIDR blocks, the auto-discovered CIDR blocks are hidden for all clusters, not only for the selected cluster in the network graph.
```
Comment (agantony) [inline modules/configure-cidr-blocks-ng20.adoc:17]:
```suggestion
When you hide the auto-discovered CIDR blocks, the auto-discovered CIDR blocks are hidden for all clusters, and not only for the selected cluster in the network graph.
```

File: modules/download-network-baselines.adoc [hunk 2]
```diff
@@ -3,13 +3,13 @@
...
-= Downloading network baselines from the network graph (2.0 preview)
+= Downloading network baselines from the network graph
 
 You can download network baselines as YAML files from the network graph view.
 
 .Procedure
-. In the {product-title-short} web portal, navigate to *Network Graph (2.0 preview)*.
-. Click the *Namespace* list and use the search field to locate a namespace or select individual namespaces.
+. In the {product-title-short} portal, navigate to *Network Graph*.
+. Click the *Namespaces* list and use the search field to locate a namespace or select individual namespaces.
```
Comment (agantony) [inline modules/download-network-baselines.adoc:12]:
```suggestion
. Click the *Namespaces* list, and use the search field to locate a namespace or select individual namespaces.
```

File: modules/enable-alert-on-baseline-violations-ng20.adoc [hunk 3]
```diff
@@ -3,13 +3,13 @@
...
 [id="enable-alert-on-baseline-violations-ng20_{context}"]
-= Enabling alerts on baseline violations in the network graph (2.0 preview)
+= Enabling alerts on baseline violations in the network graph
 
 You can configure {product-title-short} to detect anomalous network flows and trigger violations for traffic that is not in the baseline. This can help you determine if the network contains unwanted traffic before you block traffic with a network policy.
 
 .Procedure
 
-. Click the *Namespace* list and use the search field to locate a namespace or select individual namespaces.
+. Click the *Namespaces* list and use the search field to locate a namespace or select individual namespaces.
```
Comment (agantony) [inline modules/enable-alert-on-baseline-violations-ng20.adoc:12]:
```suggestion
. Click the *Namespaces* list, and use the search field to locate a namespace or select individual namespaces.
```

File: modules/generate-network-policies-ng20.adoc [hunk 4]
```diff
@@ -3,30 +3,19 @@
...
-= Generating network policies in the network graph (2.0 preview)
+= Generating network policies in the network graph
 
 [role="_abstract"]
 {product-title-short} lets you automatically generate network policies based on the actual observed network communication flows in your environment.
 
 You can generate network policies from the network graph.
 
-The generated policies apply to the deployments shown in the network graph and they allow all network traffic observed during the selected time.
+The generated policies apply to all deployments that exist in the currently-selected cluster and they allow all network traffic observed during the baseline discovery period.
```
Comment (agantony) [inline modules/generate-network-policies-ng20.adoc:13]:
```suggestion
The generated policies apply to all the deployments that exist in the currently-selected cluster and they allow all network traffic observed during the baseline discovery period.
```

File: modules/view-deployment-info.adoc [hunk 5]
```diff
@@ -18,8 +18,8 @@ The network graph provides a visual map of deployments, namespaces, and connecti
 .Procedure
 To view details for deployments in a namespace:
 
-. In the {product-title-short} portal, navigate to *Network Graph (2.0 preview)* and select your cluster from the drop-down list.
-. Click the *Namespace* list and use the search field to locate a namespace or select individual namespaces.
+. In the {product-title-short} portal, navigate to *Network Graph* and select your cluster from the drop-down list.
+. Click the *Namespaces* list and use the search field to locate a namespace or select individual namespaces.
```
Comment (agantony) [inline modules/view-deployment-info.adoc:22]:
```suggestion
. Click the *Namespaces* list, and use the search field to locate a namespace or select individual namespaces.
```

File: modules/view-network-policies-ng20.adoc [hunk 6]
```diff
@@ -3,20 +3,20 @@
...
 Kubernetes `NetworkPolicy` resources use labels to select pods and define rules that specify what traffic is allowed to or from the selected pods.
 {product-title-short} discovers and displays network policy information for all your Kubernetes clusters, namespaces, deployments, and pods, in the network graph.
 
 .Procedure
-. In the {product-title-short} portal, navigate to *Network Graph (2.0 preview)* and select your cluster from the drop-down list.
-. Click the *Namespace* list and use the search field to locate a namespace or select individual namespaces.
-. Click the *Deployments* list and use the search field to locate a deployment or select individual deployments to display in the network graph.
+. In the {product-title-short} portal, navigate to *Network Graph* and select your cluster from the drop-down list.
+. Click the *Namespaces* list and select individual namespaces, or use the search field to locate a namespace.
+. Click the *Deployments* list and select individual deployments, or use the search field to locate a deployment.
```
Comment (agantony) [inline modules/view-network-policies-ng20.adoc:16]:
```suggestion
. Click the *Deployments* list, and select individual deployments, or use the search field to locate a deployment.
```

**Review by agantony — COMMENTED** [review]
Overall LGTM. Added a few minor suggestions :-)

File: modules/apply-generated-policies-ng20.adoc [hunk 7]
```diff
@@ -1,9 +1,9 @@
...
 // * operating/manage-network-policies.adoc
-:content-type: PROCEDURE
+:_content-type: PROCEDURE
 [id="apply-generated-policies-ng20_{context}"]
-= Applying generated policies in the network graph (2.0 preview)
+= Applying generated policies in the network graph
 
 [role="_abstract"]
-You cannot apply generated network policies from the network graph in the network graph (2.0 preview). Apply Kubernetes network policies as part of your automated procedures.
+This functionality has been removed; you cannot apply generated network policies directly in the network graph. Apply Kubernetes network policies as part of your automated procedures.
```
[the remark this answers is not in the document]
Comment (snarayan-redhat) [reply modules/apply-generated-policies-ng20.adoc:9]: This content sounds like a Release Note. Do we want to revert it back to what it was?

File: modules/download-network-baselines.adoc (hunk 2 above)
Comment (snarayan-redhat) [inline modules/download-network-baselines.adoc:12]:
```suggestion
. Click the *Namespaces* list, and use the search field to locate a namespace, or select individual namespaces.
```

**Review by snarayan-redhat — COMMENTED** [review]
Couple of additional comments.

**Pull request merged**